core/capabilities/ccip/delegate.go

```diff
@@ -303,12 +303,12 @@ func bindReader(ctx context.Context,
 	capabilityLabelledName,
 	capabilityVersion string,
 ) (boundReader types.ContractReader, ccipConfigBinding types.BoundContract, err error) {
-	err = reader.Bind(ctx, []types.BoundContract{
-		{
-			Address: capRegAddress,
-			Name:    consts.ContractNameCapabilitiesRegistry,
-		},
-	})
+	binding := types.BoundContract{
```

Comment:
Should these be aligned one way or another?
```suggestion
	boundContract := types.BoundContract{
```
```suggestion
	binding := types.Binding{
```

Reply:
The type in common is `BoundContract`. If changing to `boundContract` makes the code read easier, I can do that.